travel_platform/test_full_landing/travel_platform.html

```diff
@@ -1,118 +1,125 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-	<meta charset="UTF-8">
-		<meta http-equiv="X-UA-Compatible" content="IE=edge">
-		<meta name="viewport" content="width=device-width, initial-scale=1">
-		<meta name="description" content="Booking system">
-		<meta name="keywords" content="Travel,travels,booking,tickets,hotels">
-		<title>Travel platform</title>
-	<link href="https://fonts.googleapis.com/css?family=Acme|Wendy+One" rel="stylesheet">
-	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
-	<link rel="stylesheet" type="text/css" href="css/style.css">
+  <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Is this `meta` part of previous `meta`? It looks like they are on different levels.